feature_engine/discretisation/base_discretiser.py

```diff
@@ -14,7 +14,7 @@ class BaseDiscretiser(BaseNumericalTransformer):
     """
 
     _return_object_docstring = """return_object: bool, default=False
-        Whether the the discrete variable should be returned as numeric or as
+        Whether the discrete variable should be returned as numeric or as
         object. If you would like to proceed with the engineering of the variable as if
         it was categorical, use True. Alternatively, keep the default to False.
         """.rstrip()
```

Comment:
could we add docstrings for precision? Something like this : `The precision at which to store and display the bins labels.`

Reply:
This parameter will be inherited by all discretizers, so we need to add the docstring in all files now I think.

Reply:
I understand adding it to the base discretiser, but why would we add it to other, if this parameter supposed to be private and dev-only?